### PR TITLE
Doc: Fix is_prime

### PR DESCRIPTION
**Pull request by JulienPalard**

### CODE REVIEW

File: Doc/library/concurrent.futures.rst [hunk 1]
```diff
@@ -257,6 +257,10 @@ ProcessPoolExecutor Example
        1099726899285419]
 
    def is_prime(n):
+       if n == 1:
```
Comment (vstinner) [inline Doc/library/concurrent.futures.rst]: I suggest n < 2 :-)

Comment (JulienPalard) [reply Doc/library/concurrent.futures.rst]: Maybe but I conider that one should not give us a negative number here as a parameter, it's out of range. It's like giving a float or a list or anything else.

Comment (vstinner) [reply Doc/library/concurrent.futures.rst]: 0 is positive and is not prime :-)

Comment (JulienPalard) [reply Doc/library/concurrent.futures.rst]: ok ok (yet not natural þ)

**Review by vstinner — APPROVED** [review]
LGTM.

**Pull request merged**